tests/check_xcode.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Script that checks if the XCode (MacOS) project is complete.
+# If the XCode project is incomplete, then this script also uses
+# mod-pbxproj (https://github.com/kronenthaler/mod-pbxproj) to
+# suggest fixes/additions to make the XCode project complete.
+#
+# mod-pbxproj should already have been installed (through PIP) before
+# running this script.
+
+
+# Determine path of the current script
+HERE=$(cd `dirname $0` && pwd)
+
+# Go to toplevel ES directory
+cd ${HERE}
```

Comment:
`cd "${HERE}/.."`?

I'm assuming this is so that we can run the test from a shell that is in any working directory, e.g. `$ ../../path/to/testfile.sh`

Reply:
Yes, that is indeed what this is for.